### PR TITLE
chore: add Nirbhay Chaplot

### PR DESCRIPTION
**Pull request by nirbhay12345**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/922"></a>

### CODE REVIEW

File: public/data/nirbhay12345.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "url": "https://github.com/nirbhay12345",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Nirbhay00704167",
+            "icon": "twitter"
+        }
+    ]
+}
```
Comment (syedsohan) [inline public/data/nirbhay12345.json:18]: Missing an end line.

```suggestion
}

```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/nirbhay12345.json (hunk 1 above)
Comment (nirbhay12345) [reply public/data/nirbhay12345.json]: I have done the changes

**Review by syedsohan — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

File: public/data/nirbhay12345.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
```
Comment (krishguptadev) [inline public/data/nirbhay12345.json:18]: Not formatted correct.

```suggestion
{
  "name": "Nirbhay Chaplot",
  "type": "personal",
  "bio": "Open Source Enthusiast!",
  "avatar": "https://github.com/nirbhay12345.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/nirbhay12345",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/Nirbhay00704167",
      "icon": "twitter"
    }
  ]
}

```

Comment (Panquesito7) [reply public/data/nirbhay12345.json:18]: Great catch, thanks! Looks like the bot has already done that for us :) Check https://github.com/EddieHubCommunity/LinkFree/commit/6e8e6621836caf631eaa76d6a93a4e65cdae2d98

Comment (nirbhay12345) [reply public/data/nirbhay12345.json:18]: Thank you @krishguptadev for spotting the misalignment. Will keep in mind next time

**Pull request merged**